server/graphql/v2/mutation/AddedFundsMutations.ts

```diff
@@ -0,0 +1,352 @@
+import assert from 'assert';
```

Comment:
The rename makes it difficult to understand what changed in the file, and I'm not 100% convinced that the new name is any clearer. I would keep that for a separate PR.

Reply:
I've separated the rename commit from the edits so you can isolate the diff:
https://github.com/opencollective/opencollective-api/pull/10241/commits/97262fd3c084f8774341996ae96198f73fe48574#diff-8b9a3f71fc3209ae45f2ce922a6f80cda530d1c1580574ce9e1603fc58fdb90

I renamed it because this file was a single mutation with the same file name.